packages/xchain-litecoin/src/client.ts

```diff
@@ -41,18 +43,20 @@ class Client implements LitecoinClient, XChainClient {
   private net: Network
   private phrase = ''
   private sochainUrl = ''
-  private bitapsUrl = ''
+  private nodeUrl = ''
+  private nodeAuth: NodeAuth
 
   /**
    * Constructor
    * Client is initialised with network type
    *
    * @param {LitecoinClientParams} params
    */
-  constructor({ network = 'testnet', sochainUrl, bitapsUrl, phrase }: LitecoinClientParams) {
+  constructor({ network = 'testnet', sochainUrl, phrase, nodeUrl, nodeAuth }: LitecoinClientParams) {
     this.net = network
+    this.nodeUrl = !!nodeUrl ? nodeUrl : this.getDefaultNodeUrl()
+    this.nodeAuth = nodeAuth || this.getDefaultNodeAuth()
     this.setSochainUrl(sochainUrl || this.getDefaultSochainUrl())
```

Comment:
Remove `getDefaultSochainUrl` and put default values into `sochainUrl` of constructor.